spec/integration/client/client_spec.rb

```diff
@@ -341,7 +342,8 @@ class ::Blah
     end
   end
 
-  context "when using recipe-url" do
+  # Fails on appveyor, but works locally on windows and on windows hosts in Ci.
+  context "when using recipe-url", :skip_appveyor do
```

Comment:
It would be awesome to not need this.  Maybe updating Rubygems will magically fix this issue?